clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/data/InstanceFamilyUtils.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.netflix.spinnaker.clouddriver.aws.data;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import java.util.Map;
+import java.util.Set;
+
+public class InstanceFamilyUtils {
+
+  private static final Map<String, Set<String>> KNOWN_VIRTUALIZATION_FAMILIES =
+      ImmutableMap.of(
+          "paravirtual", ImmutableSet.of("c1", "c3", "hi1", "hs1", "m1", "m2", "m3", "t1"),
+          "hvm", ImmutableSet.of("c3", "c4", "d2", "i2", "g2", "r3", "m3", "m4", "t2", "t3"));
```

Comment:
this doesn't seem like a comprehensive list
would it be safe to check specifically for paravirtual since that list is probably unchanging at this point and maybe metal if that isn't still surfaced through hvm and otherwise assume hvm is acceptable?

Reply:
Yes, I think that makes sense. Will make the change.